diff --git a/anki/collection.py b/anki/collection.py
index 6cec228..189c706 100644
--- a/anki/collection.py
+++ b/anki/collection.py
@@ -60,7 +60,7 @@ class _Collection:
         self.server = server
         self._lastSave = time.time()
         self.clearUndo()
-        self.media = MediaManager(self, server)
+        self.media = MediaManager(self)
         self.models = ModelManager(self)
         self.decks = DeckManager(self)
         self.tags = TagManager(self)
diff --git a/anki/media.py b/anki/media.py
index 4e3ccf3..3bfdc81 100644
--- a/anki/media.py
+++ b/anki/media.py
@@ -26,11 +26,8 @@ class MediaManager:
     ]
     regexps = soundRegexps + imgRegexps
 
-    def __init__(self, col, server):
+    def __init__(self, col):
         self.col = col
-        if server:
-            self._dir = None
-            return
         # media directory
         self._dir = re.sub("(?i)\.(anki2)$", ".media", self.col.path)
         if not os.path.exists(self._dir):
@@ -48,8 +45,6 @@ class MediaManager:
         self.connect()
 
     def connect(self):
-        if self.col.server:
-            return
         path = self.dir()+".db2"
         create = not os.path.exists(path)
         os.chdir(self._dir)
@@ -100,8 +95,6 @@ create table meta (dirMod int, lastUsn int); insert into meta values (0, 0);
             os.rename("../collection.media.db", npath)
 
     def close(self):
-        if self.col.server:
-            return
         self.db.close()
         self.db = None
         # change cwd back to old location
